HACKING.md

```diff
@@ -119,7 +119,7 @@ $ ./test.py --mode={debug,release}
 or, if you are using `dbuild`, you need to build the tests and then you can run them at will:
 
 ```console
-$ ./tools/toolchain/dbuild ninja {debug,release,dev}-test
+$ ./tools/toolchain/dbuild ninja {debug,release,dev}-build
```

Comment:
I'm confused now... Why does dev-build and dev-test both exist, if they both compile tests?
Did you test that "ninja dev-build" actually builds all the test executables?

Reply:
dev-build builds all the test executables, and scylla itself.

dev-test builds only the test executables.

I think *test should be retired, since it's impossible to test without the scylla executable.

Reply:
I did test - dev-build builds all tests and I can run ./test.py with all tests (no filtering) afterwards. I could not before that.